docs/administration/upgrading/guide/creating-test-instance.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Creating a test instance
+description: A guide on how to create a test instance with Octopus Deploy
+position: 8
+---
+
+There is always a risk an in-place upgrade will fail.  The risk increases further away from the latest version.  We do our best to test as many different configurations as to put downward pressure on that risk. The truth of the matter is we can't cover every hyper-specific scenario.  Also, there might be a new feature, or a breaking change introduced.  Creating a test instance will help reduce the risk for you and your company.
+
+## Overview
+
+There are two approaches to creating a test instance.
+
+1. Subset of projects representing the main instance
+1. Clone of the main instance
+
+### Test instance with a subset of projects
+
+Setting up a test instance with a subset of projects over a full clone has several advantages.  
+
+- It is much easier to set up and configure than a clone.  It is also easy to automate the setting up and tear down a test instance on demand.
+- There is tooling in place to export specific projects.
+- A clone does a full clone of everything, including deployment targets and triggers, increasing the risk of having the sandbox instance connect to production at first.
```

Comment:
I'm not sure what `at first` is referring to.

Reply:
One for @OctopusBob, but perhaps this should say `when started`?

Context: I _think_ the point is that cloning an instance carries an increased risk of connecting to production when you start that cloned instance (since it will think its the main instance) without remedial scripts to change the state of the cloned instance